substrate-processor/src/batch.ts

```diff
@@ -18,13 +27,25 @@ export interface DataHandlers {
     evmLogs: Record<EvmContractAddress, {filter?: EvmTopicSet[], handler: EvmLogHandler}[]>
 }
 
-
+/**
+ * Defines a batch of blocks to be processed
+ * 
+ * @property range: a {@link Range} defining start and end blocks for the batch
+ * @property handlers: a {@link DataHandlers} objects to collect the handlers that should be triggered for the batch.
+ */
 export interface Batch {
     range: Range
     handlers: DataHandlers
 }
 
-
+/**
+ * Forms an array of {@link Batch} for the given {@link Range} by mixing in the provided hooks
```

Comment:
This is a bit confusing as it's not clear what 'mixing' really means. Smth like `Extracts a Batch of handlers that should be executed within a given block Range`